json-server/index.js

```diff
@@ -0,0 +1,32 @@
+const jsonServer = require('json-server');
+const { apiPath } = require('../config/env/development');
+
+module.exports = {
+  initialize(server) {
+    const router = jsonServer.router('./json-server/db/db.json');
+
+    server.use(jsonServer.defaults());
+    server.use(apiPath, router);
+
+    // Has overrided json-server render method to simulate server side error response.
+    // Error will be returned if you try to sign in with error@example.com email
+    router.render = (
+      {
+        url,
+        method,
+        body: { email }
+      },
+      res
+    ) => {
+      if (
+        url === '/session' &&
+        method === 'POST' &&
+        email === 'error@example.com'
+      ) {
+        res.status(500).jsonp({ error: 'Server error has occured' });
+      } else {
+        res.jsonp(res.locals.data);
```

Comment:
JSONP? For what? Are you sure that this will compatible with Axios configuration on client-side?

Reply:
It is default override for json-server render method. No any rocket science here. You can see it here https://github.com/typicode/json-server. And ofc it works with axios.